PROJECT

```diff
@@ -13,4 +17,13 @@ resources:
   kind: Space
   path: github.com/edixos/nauticus/api/v1alpha1
   version: v1alpha1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: nauticus.io
+  group: nauticus.io
```

Comment:
The group should be empty I think for next generation.

Reply:
for now, we keep it like it's?